teacher.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'person'
+class Teacher < Person
+  def initialize(name, age, parent_permission, specialization = 'Computer Science')
```

Comment:
- [ ] I think you don't need to give the specialization a value, the users will give themselves value to the specialization if it's computer science or Biology,... 🙃 
- [ ] Also the name has the value `'Unknown'` and you should set your parent_permission to `true` 😉 😉